src/components/modules/dragNDrop.ts

```diff
@@ -75,16 +65,43 @@ export default class DragNDrop extends Module {
 
     dropEvent.preventDefault();
 
+    if (BlockManager.currentDraggingBlock) {
```

Comment:
maybe you could move this part of code to the separate method? Because there are many lines of code